examples/research_projects/stack_llama_2/scripts/dpo_llama2.py

```diff
@@ -176,6 +177,7 @@ def return_prompt_and_responses(samples) -> Dict[str, str]:
         remove_unused_columns=False,
         run_name="dpo_llama2",
     )
+    training_args.gradient_checkpointing_kwargs = dict(use_reentrant=False)
```

Comment:
Can you make that configurable through `script_args` so that users can disable it if they want?

Reply:
I have create a new pr https://github.com/huggingface/trl/pull/1353